sdk/eventhub/event-hubs/src/util/operationOptions.ts

```diff
@@ -0,0 +1,31 @@
+import { AbortSignalLike } from '@azure/abort-controller';
+import { SpanOptions } from "@azure/core-tracing";
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
I _think_ these need to be the first lines in the file.

Reply:
Yes...I think VSCode re-org'd that for me.